src/lib/coreboot_table.c

```diff
@@ -403,6 +405,45 @@ void __weak lb_board(struct lb_header *header) { /* NOOP */ }
  */
 void __weak lb_spi_flash(struct lb_header *header) { /* NOOP */ }
 
+/*
```

Comment:
Does this work? I was under the impression this code won't work on newer revisons of coreboot. That's why this was rewritten as I mentioned here: https://github.com/Dasharo/coreboot/pull/352

Reply:
Yup it still works, but I will apply your change here.

Reply:
@mkopec fixed by cherry-picking https://github.com/Dasharo/coreboot/pull/355/commits/4d866ff8ffabe30624eaf380872ee2850b82ac2c